package.json

```diff
@@ -1,62 +1,47 @@
 {
   "name": "@wibrc/eslint-config-vue",
   "version": "0.0.1",
-  "description": "reusable and sharable `eslint` configuration",
+  "description": "reusable and sharable `eslint` configuration for vue project",
   "main": "index.js",
   "scripts": {
     "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore",
+    "lint:fix": "eslint --fix",
     "stylelint": "stylelint '**/*.css'",
     "get-version": "node -p \"require('./package').version\""
   },
   "keywords": [
     "eslint",
+    "prettier",
+    "vue",
+    "stylelint",
     "configuration"
   ],
   "author": "wibrc",
   "license": "ISC",
   "peerDependencies": {
-    "eslint": "~8.32.0",
-    "eslint-config-airbnb-base": "~15.0.0",
-    "eslint-config-prettier": "~8.6.0",
-    "eslint-plugin-import": "~2.27.5",
-    "eslint-plugin-prettier": "~4.2.1",
-    "prettier": "~2.8.3",
-    "@vue/eslint-config-prettier": "~7.0.0",
-    "@vue/eslint-config-typescript": "~10.0.0",
-    "eslint-plugin-cypress": "~2.12.1",
-    "eslint-plugin-storybook": "~0.5.12",
-    "eslint-plugin-vue": "~8.2.0",
-    "stylelint": "~14.2.0",
-    "stylelint-config-html": "~1.0.0",
-    "stylelint-config-recommended": "~6.0.0",
-    "stylelint-config-recommended-vue": "~1.1.0",
-    "stylelint-config-standard": "~24.0.0",
-    "stylelint-config-standard-scss": "~3.0.0",
-    "stylelint-processor-html": "~1.0.0",
-    "stylelint-scss": "~4.1.0",
-    "lint-staged": "12.1.2"
+    "lint-staged": "12.1.2",
+    "prettier": "2.8.3",
+    "stylelint": "14.2.0",
+    "eslint": "8.32.0"
   },
   "dependencies": {
-    "eslint": "~8.32.0",
+    "@rushstack/eslint-patch": "~1.1.0",
```

Comment:
I think you can change `~` into `^` here.

Reply:
I tried to show the difference and i found this: https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json#comment53973834_22345808.

That is not a bad idea but i'm going to think about that.